asg_util_alarms.py

```diff
@@ -6,21 +8,22 @@
 
 # Parse arguments
 parser = ArgumentParser()
-parser.add_argument('boundaries', metavar=('cpu lower', 'cpu upper', 'disk lower', 'disk upper'), type=int, nargs=4,
-                    help="Supply cpu lower, cpu upper, disk lower, disk upper bound (0-100)")
+parser.add_argument('boundaries', metavar=('cpu upper', 'disk upper'), type=int, nargs=2,
+                    help="Supply cpu upper, disk upper bound (0-100)")
 args = parser.parse_args()
 
-cpu_lower = args.boundaries[0]
-cpu_upper = args.boundaries[1]
-disk_lower = args.boundaries[2]
-disk_upper = args.boundaries[3]
+cpu_upper = args.boundaries[0]
+cpu_lower = cpu_upper * 0.8
```

Comment:
We should consider using 0.5 (or lower) so that a scale-up does not trigger an immediate scale-down after the system goes to a steady state.

Reply:
updated!